### PR TITLE
Correct `kubectl operator install` for latest version of operator-sdk

### PR DESCRIPTION
**Pull request by ElanHasson**
Fixes https://github.com/cert-manager/website/issues/981

### CODE REVIEW

File: content/docs/installation/operator-lifecycle-manager.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ from the [Krew Kubectl plugins index][] and then use that to install the cert-ma
 ```sh
 operator-sdk olm install
 kubectl krew install operator
-kubectl operator install cert-manager -n operators --channel stable --approval Automatic
+kubectl operator install cert-manager -n cert-manager --channel candidate --approval Automatic --create-operator-group
```
Comment (wallrj) [inline content/docs/installation/operator-lifecycle-manager.md:44]: This didn't work for me without first creating the cert-manage ns.
And we should continue to recommend the `stable` channel.
The `candidate` channel contains test packages. 


```bash
kubectl create ns cert-manager
kubectl operator install cert-manager -n operators --channel stable --approval Automatic --create-operator-group -n cert-manager
```

```
operatorgroup "cert-manager" created
subscription "cert-manager" created
operator "cert-manager" installed; installed csv is "cert-manager.v1.11.4"
```

**Review by wallrj — CHANGES_REQUESTED** [review]

**Comment by ElanHasson** [comment]
Updated @wallrj 

Sorry for the delay, just saw this.

File: content/docs/installation/operator-lifecycle-manager.md (hunk 1 above)
Comment (ElanHasson) [reply content/docs/installation/operator-lifecycle-manager.md]: Excellent points. Late night drive-by docs PRs to get stuff installed in the homelab 😅

File: content/docs/installation/operator-lifecycle-manager.md [hunk 2]
```diff
@@ -41,7 +41,8 @@ from the [Krew Kubectl plugins index][] and then use that to install the cert-ma
 ```sh
 operator-sdk olm install
 kubectl krew install operator
-kubectl operator install cert-manager -n operators --channel stable --approval Automatic
+kubectl create ns cert-manager
+kubectl operator install cert-manager -n operators --channel stable --approval Automatic --create-operator-group -n cert-manager
```
Comment (wallrj) [inline content/docs/installation/operator-lifecycle-manager.md:45]:
```suggestion
kubectl operator install cert-manager -n cert-manager --channel stable --approval Automatic --create-operator-group
```

**Review by wallrj — CHANGES_REQUESTED** [review]
I suggested one change and I tested that on a kind cluster, as follows:

```
kind create cluster
./bin/operator-sdk-1.25.0 olm install
kubectl krew upgrade

$ kubectl krew list
PLUGIN         VERSION
access-matrix  v0.5.0
krew           v0.4.4
operator       v0.4.0
who-can        v0.4.0

kubectl create ns cert-manager
kubectl operator install cert-manager --channel stable --approval Automatic --create-operator-group -n cert-manager
```

Then I ran some of the cert-manager E2E tests to make sure the installation works:

```
~/projects/cert-manager/cert-manager$ make e2e-build
~/projects/cert-manager/cert-manager$  ./_bin/test/e2e.test --repo-root=/dev/null --ginkgo.focus="CA\ Issuer" --ginkgo.skip="Gateway"
Running Suite: cert-manager e2e suite - /home/richard/projects/cert-manager/cert-manager
========================================================================================
Random Seed: 1698831593

Will run 44 of 778 specs
"hashicorp" already exists with the same configuration, skipping
Release "chart-vault-vault" does not exist. Installing it now.
NAME: chart-vault-vault
LAST DEPLOYED: Wed Nov  1 09:39:55 2023
NAMESPACE: e2e-vault
STATUS: deployed
REVISION: 1
NOTES:
Thank you for installing HashiCorp Vault!

Now that you have deployed Vault, you should look over the docs on using
Vault with Kubernetes available here:

https://www.vaultproject.io/docs/


Your release is named chart-vault-vault. To learn more about the release, try:

  $ helm status chart-vault-vault
  $ helm get manifest chart-vault-vault
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS••[controller-runtime] log.SetLogger(...) was never called; logs will not be displayed.
Detected at:
        >  goroutine 268 [running]:
        >  runtime/debug.Stack()
        >       runtime/debug/stack.go:24 +0x5e
        >  sigs.k8s.io/controller-runtime/pkg/log.eventuallyFulfillRoot()
        >       sigs.k8s.io/controller-runtime@v0.16.2/pkg/log/log.go:60 +0xcd
        >  sigs.k8s.io/controller-runtime/pkg/log.(*delegatingLogSink).WithName(0xc000242040, {0x1da9091, 0x14})
        >       sigs.k8s.io/controller-runtime@v0.16.2/pkg/log/deleg.go:147 +0x45
        >  github.com/go-logr/logr.Logger.WithName({{0x2073af0, 0xc000242040}, 0x0}, {0x1da9091?, 0xc000aafd98?})
        >       github.com/go-logr/logr@v1.2.4/logr.go:336 +0x3d
        >  sigs.k8s.io/controller-runtime/pkg/client.newClient(0x0?, {0x0, 0xc00011b110, {0x0, 0x0}, 0x0, {0x0, 0x0}, 0x0})
        >       sigs.k8s.io/controller-runtime@v0.16.2/pkg/client/client.go:122 +0xec
        >  sigs.k8s.io/controller-runtime/pkg/client.New(0x1dc379a?, {0x0, 0xc00011b110, {0x0, 0x0}, 0x0, {0x0, 0x0}, 0x0})
        >       sigs.k8s.io/controller-runtime@v0.16.2/pkg/client/client.go:103 +0x7d
        >  github.com/cert-manager/cert-manager/e2e-tests/framework.(*Framework).BeforeEach(0xc00023d040)
        >       github.com/cert-manager/cert-manager/e2e-tests/framework/framework.go:145 +0x33c
        >  github.com/onsi/ginkgo/v2/internal.extractBodyFunction.func3({0xa01dc8, 0xc0008c7500})
        >       github.com/onsi/ginkgo/v2@v2.12.0/internal/node.go:463 +0x13
        >  github.com/onsi/ginkgo/v2/internal.(*Suite).runNode.func3()
        >       github.com/onsi/ginkgo/v2@v2.12.0/internal/suite.go:865 +0x8d
        >  created by github.com/onsi/ginkgo/v2/internal.(*Suite).runNode in goroutine 16
        >       github.com/onsi/ginkgo/v2@v2.12.0/internal/suite.go:852 +0xd7b
••SSS••••••••••••••••••SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSproxy logs:


Ran 23 of 778 Specs in 72.842 seconds
SUCCESS! -- 23 Passed | 0 Failed | 0 Pending | 755 Skipped
PASS
```

**Review by wallrj — APPROVED** [review]
/lgtm

**Pull request merged**